release/mk-langpack.sh

```diff
@@ -13,7 +13,13 @@ webpath=`cat ~/.website-path`
 filedir=${webpath}/content/Versions/langpack
 
 # Location of the repository checkout
-vcsdir=/srv/sources/squid-master
+vcspath=`cat ~/.code-vcs-path`
+vcsver=`ls -1 $vcspath | cut -d- -f2 | sort -h`
+vcsdir="${vcspath}squid-${vcsver}"
+if ! test -d $vcsdir ; then
```

Comment:
I do not know enough to request specific improvements (or even to know where those improvements should be made), but please note that this change is inconsistent with `release/mk-release-series.sh` being imported in #8. The latter code calls the contents of the `~/.code-vcs-path` file "vcsdir" rather than "vcspath" and has no vcspath-to-vcsdir adjustments present in this PR above:

```sh
# Location of the repository checkouts
vcsdir=`cat ~/.code-vcs-path`
```

In other words, vcsdir here is different from vcsdir in #8 despite the same name and shared association with `~/.code-vcs-path`.

Reply:
I think this change request has merit, we should be consistent if we define this internal API

Reply:
Done as of [4b0aaab](https://github.com/squid-cache/ci/pull/7/commits/4b0aaab12a3e24000471ab2a5030ba3e1452b87f)